### PR TITLE
Disable `ErrorBar` in `ChatComposite` by default

### PR DESCRIPTION
**Pull request by prprabhu-ms**
# What
Disable `ErrorBar` in `ChatComposite` and add a feature flag to enable it.
Also enable it in automated tests, so that `ErrorBar` tests continue to provide coverage.

# Why
Error reporting work is still WIP so we don't want to include it in the next beta release.

# How Tested
Automated tests with/without feature flag to enable the ErrorBar.

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by prprabhu-ms** [comment]
Only adds an optional flag, so not a breaking change.

Now when I do remove that flag... that will be breaking ;)

File: packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx [hunk 1]
```diff
@@ -34,6 +34,14 @@ export type ChatCompositeProps = {
  * Additional customizations for the chat composite
  */
 export type ChatOptions = {
+  /**
+   * UNSTABLE: Feature flag to enable ErrorBar.
+   *
+   * This option will be removed once ErrorBar is stable.
```
Comment (anjulgarg) [inline packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx:40]: IMO even when the error bar becomes stable, it's good to have it as an optional. CONTOSO might not want to show the error bar...

File: packages/storybook/stories/ErrorBar/ErrorBar.stories.tsx [hunk 2]
```diff
@@ -25,6 +25,10 @@ const getDocs: () => JSX.Element = () => {
         `ErrorBar` is a wrapper on fluent UI's `MessageBar` with additional features for surfacing Azure Communication
         Services errors on the UI consistently.
       </Description>
+      <Description>
+        Set the `showErrorBar` option for `ChatComposite` to use an `ErrorBar` to show errors. This feature is still
```
Comment (anjulgarg) [inline packages/storybook/stories/ErrorBar/ErrorBar.stories.tsx:29]: I would recommend removing the `This feature...` line from here.
We are still in Beta and this is a given anyway.
Plus we might forget to remove this piece of text once ErrorBar becomes stable.

**Review by anjulgarg — APPROVED** [review]

File: packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx:40]: Could also tag it `@experimental` https://tsdoc.org/pages/tags/experimental/

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx (hunk 1 above)
Comment (prprabhu-ms) [reply packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx:40]: Tagged it @experimental.

Not sure we want to keep this option later on, so playing it safe.

**Pull request merged**